dockerfiles/hpcc/templates/roxie.yaml

```diff
@@ -82,10 +103,21 @@ spec:
                 {{ include "hpcc.configArg" . }},
                 {{ include "hpcc.daliArg" $ }},
                 "--numChannels={{ $numChannels }}",
-                "--serverPorts={{ template "hpcc.portListToCommas" $roxie.ports }}", 
+                "--server=true", 
                 "--topologyServers={{ $toponame }}:{{ $roxie.topoport }}",
                 "--resolveLocally=false"
               ]
+{{- range $service := $roxie.services }}
+{{- $local := dict "first" true }}
```

Comment:
Shouldn't this be outside the range, otherwise ports will appear more than once?

Reply:
Yes, probably. Should also be similar code in localroxie.yaml - though I think its absence is actually pretty harmless.